src/components/KonnectorTile.jsx

```diff
@@ -97,7 +114,8 @@ const mapStateToProps = (state, props) => {
     error: getFirstError(state.connections, konnector.slug),
     userError: getFirstUserError(state.connections, konnector.slug),
     lastSyncDate: getLastSyncDate(state.connections, konnector.slug),
-    accountsCount: getKonnectorTriggersCount(state, konnector)
+    accountsCount: getKonnectorTriggersCount(state, konnector),
+    inMaintenance: getKonnectorsInMaintenance()[konnector.slug]
```

Comment:
`inMaintenance: isInMaintenance(konnector.slug)` is more consistent with redux concepts, `isInMaintenance` must be implemented, based on `getKonnectorsInMaintenance()`.

Reply:
This seems unresolved to me, we need here something like:
```
inMaintenance: isKonnectorInMaintenance(konnector.slug)
```